recipe/meta.yaml

```diff
@@ -10,22 +10,21 @@ source:
   sha256: 8a3438333a8d290290a3a0962a8de64bc6e191c79ac6970e8e2b36cc6558ff26
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
     - pip
-    - python >=3.7
+    - python
     - setuptools
     - setuptools_scm
   run:
     - antlr-python-runtime =4.9.2
     - pyodbc >=4.0.30
-    - python >=3.7
```

Comment:
Why did you remove the lower restriction for Python?

Reply:
And then, do you know how to convey a certain python version requirement then?

Reply:
Not sure, I'm looking into it

Reply:
Just added an option that should be equivalent to the restriction we had before:

https://github.com/conda-forge/pytsql-feedstock/pull/5/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR15